keras_cv/utils/fill_utils.py

```diff
@@ -16,15 +16,16 @@
 from keras_cv.utils import bbox
 
 
-def rectangle_masks(mask_shape, corners):
-    """Computes positional masks of rectangles in images
+def rectangle_masks(corners, width, height):
+    """Computes masks of rectangles
 
     Args:
-        mask_shape: shape of the masks as [batch_size, height, width].
         corners: rectangle coordinates in corners format.
```

Comment:
I think its fine to just use mask_shape as the param (which is in shape [batch, width, height]). The width and height usually should be combined together as one param.

One a side note, please add shape/rank infrmation for the docstring.

Reply:
The `batch_size` of the masks, is determined by the number of bboxes in `corners` argument, so I think `mask_shape` should only be [width, height] then.